src/components/common/TmSessionExplore.vue

```diff
@@ -67,7 +67,7 @@ export default {
         address: this.address
       })
       localStorage.setItem(`prevAddress`, this.address)
-      this.$router.push(`/`)
+      this.$router.push(`/portfolio`)
```

Comment:
please change those back. `/` forwards to `/portfolio`. This way we can change the default route easier in the router.